marketplace/README.md

```diff
@@ -0,0 +1,186 @@
+# Setting Up for Marketplace Solution
```

Comment:
Can you qualify "Marketplace" so it is clear it's GCP Marketplace (or whatever the official name)?

Reply:
ack.